adapter/internal/ga/client.go

```diff
@@ -239,6 +241,24 @@ func addAPIToChannel(resp *discovery.DiscoveryResponse) {
 			if currentGAAPI.RevisionUUID == api.RevisionUUID {
```

Comment:
We should check revisionUUID after processing the timestamp.
if the timestamp < hardCodedValue , then we should use revisionUUID alone to do the comparison. 
else we can assign the whole revisionUUID (with timestamp and move on)

Reply:
Fixed